overlays/non-privileged/README.md

```diff
@@ -13,5 +13,5 @@ To use it, execute the following command from the root directory of this reposit
 After executing the script you can apply the generated manifests from the `generated-cluster` directory:
 
 ```shell script
-kubectl apply --prune -l deploy=sourcegraph -f generated-cluster --recursive
```

Comment:
Will this cause issues on update for any customers who previously omitted the namespace? If so, how will we address that?

Reply:
no issue. it works without. i just wanted to make it explicit that they are working in a namespace in case they use kubectl to do log/describe and other commands. doing apply works because the namespace is declared in the manifests.